.circleci/config.yml

```diff
@@ -501,6 +514,17 @@ jobs:
             - /tmp/go-cache
       - store_artifacts:
           path: bin/linux
+      - persist_to_workspace:
```

Comment:
If we might bail directly after this, why not wait to persist artifacts to the workplace until after? Also, I believe that the objects persisted here can be rolled into the `persist_to_workspace` stanza four steps below this one.

Reply:
My understanding, btw, is that `persist_to_workspace` is only for passing items to follow-on jobs, and isn't the same as `store_artifacts`, which allows a user to inspect the artifact after the job is done.

Reply:
oh nevermind, I get it. With exit 0 we continue to follow-on jobs.